### DISCUSSION TITLE
Temperature of 127...!

### DISCUSSION THREAD

**Discussion by hirael** [tcgoetz/GarminDB#158] [Q&A]
Hello!  Can anyone please advise why I am getting a temp of 127 for some (not all) of my Yoga activities?  I have tried looking at the log file (FiT and Log should be Attached), but, it seems that the temperature entries are invalid.  Which is what I'd expect as I don't think the vivoactive3 supports temperature?

Thank you :)

[5465002201_ACTIVITY.zip](https://github.com/tcgoetz/GarminDB/files/9285005/5465002201_ACTIVITY.zip)

**Comment by tcgoetz** [comment]
I see this in the parsed FIT file:

avg_temperature([invalid] (Temperature(invalid))), max_temperature([invalid] (Temperature(invalid))

So, if your watch supports temp, it isn't writing it to the FIT file. Maybe a stetting on the watch?

**Comment by hirael** [comment]
Thank you for getting back to me...  Having re-looked at the sql database, there are 366 entries with a temp of 127.  The activities are running, walking, cycling, elliptical and yoga.

Apparently, the watch has a temperature sensor as part of the  barometer and compass and is used in combination with the barometer to calculate altitude. But it isn't available in data fields or saved in the activity files.

Other than the 366 entries of 127degs - none of the other activities recorded with the watch have a temperature entry.

**Comment by hirael** [comment]
avg_temperature 127C  & max_temperature 127C are present in the CSV file produced by FitCSVTool.jar when viewed using -s option (show invalid fields).  It looks like it has been going on since Aug 2019, I'll try a factory-reset on the watch but there is probably not much else I can do...
